src/rime/dict/dict_compiler.cc

```diff
@@ -231,15 +231,15 @@ bool DictCompiler::BuildTable(int table_index,
   Vocabulary vocabulary;
   // build .table.bin
   {
-    map<string, SyllableId> syllable_to_id;
+    hash_map<string, SyllableId> syllable_to_id;
```

Comment:
Not 100% sure: the build result may rely on the fact the map keys is ordered alphabetically.
I need to take a closer look at all the changes to `hash_map`.

Reply:
> Not 100% sure: the build result may rely on the fact the map keys is ordered alphabetically. I need to take a closer look at all the changes to `hash_map`.

I think we don't need to worry here. The `hash_map` is created there in place and use key (`operator[]` programmatically) to store and access the value, not by the order.